python/cudnn/linear_attention/cutile/kernels/common.py

```
@@ -109,18 +82,185 @@ def ensure_cuda_context(stream=0) -> None:
+            with ct.compiler_timeout(timeout):
+                result = exhaustive_search(
+                    configs,
+                    stream,
+                    lambda cfg: grid,
+                    kernel,
+                    lambda cfg: args,
+                    lambda cfg: {"occupancy": cfg.occupancy, "num_worker_warps": cfg.num_worker_warps},
+                )
+            best = result.best.config
+            tuned = kernel.replace_hints(occupancy=best.occupancy, num_worker_warps=best.num_worker_warps)
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)